tidb-cloud/import-with-mysql-cli.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Import Data into TiDB Cloud via MySQL Commands in Terminal
+summary: Learn how to import Data into TiDB Cloud via MySQL CLI.
+---
+
+# Import MySQL Data via MySQL Commands in Terminal
+
+## Prerequisites:
+
+Before you can import MySQL data to TiDB Cloud, you should have the following prerequisites in place:
+
+- Access to your TiDB Cloud Cluster.
+- Necessary connection details obtained from the TiDB Cloud portal.
+- An initial password created for your TiDB user (if not already done).
+- TiDB cluster connection information.
```

Comment:
Can we move the mysql-client installation here as prerequisites?

Reply:
For this part, we only need our audience to create a `TiDB Cloud Cluster` and install `MySQL Client` beforehand.
You can get some pre-written chunk form:
- TiDB Cloud Cluster create steps: https://github.com/pingcap/docs/blob/master/develop/dev-guide-gui-datagrip.md?plain=1#L21-L43
- MySQL Client installation: https://github.com/pingcap/docs/blob/master/develop/dev-guide-build-cluster-in-cloud.md?plain=1#L62-L131